File: trunk/ruby/GrabzIt/lib/grabzitcookie.rb
class GrabzItCookie
	def initialize
		@Name = ''
		@Value = ''
		@Domain = ''
		@Path = ''
		@HttpOnly = false
		@Expires = nil		
		@Type = nil	
	end
	def name
		@Name
	end
	def name=(value)
		@Name = value
	end  
	def value
		@Value
	end
	def value=(value)
		@Value = value
	end  
	def domain
		@Domain
	end
	def domain=(value)
		@Domain = value
	end  
	def path
		@Path
	end
	def path=(value)
		@Path = value
	end  
	def httpOnly
		@HttpOnly
	end
	def httpOnly=(value)
		@HttpOnly = value
	end  
	def expires
		@Expires
	end
	def expires=(value)
		@Expires = value
	end  	
	def type
		@Type
	end
	def type=(value)
		@Type = value
	end  	
end

File: trunk/python/GrabzIt/GrabzItCookie.py
#!/usr/bin/python

class GrabzItCookie:

	def __init__(self, name, value, domain, path, httponly, expires, type):
		self.Name = name
		self.Value = value
		self.Domain = domain
		self.Path = path
		self.HttpOnly = httponly
		self.Expires = expires
		self.Type = type